test_scripts/Polices/Validation_of_PolicyTables/228_ATF_preloaded_pt_does_not_exists.lua

```diff
@@ -103,15 +106,13 @@ end
 
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
+function Test.TestStep_start_sdl()
+  StartSDL(config.pathToSDL, config.ExitOnCrash)
+  os.execute("sleep 5")
+end
 
 function Test:TestStep_checkSdl_Running()
-  --In case SDL stops function will return true
-  local result = testCasesForPolicySDLErrorsStops:CheckSDLShutdown(self)
-  if (result ~= true) then
-    self:FailTestCase("Error: SDL is running without access to preloaded_pt.json.")
-  else
-    print("SDL stopped")
-  end
+  testCasesForExternalUCS.checkSDLStatus(self, sdl.STOPPED)
 end
 
 function Test:TestStep_CheckSDLLogError()
```

Comment:
@istoimenova, Please add check on "BasicCommunication.OnSDLClose" in order to conform to other scripts